core/src/types/operator/operator.rs

```diff
@@ -831,7 +831,7 @@ impl Operator {
         Ok(Lister::new(pager))
     }
 
-    /// List dir in flat way.
+    /// Iterate through all files inside a directory bottom-up.
```

Comment:
This comment is wrong.

Reply:
What is the accurate description? `flat` seems obscure

Reply:
> What is the accurate description? `flat` seems obscure

I don't have any ideas yet, and this change is not relevant to this PR. Let's undo this modification.